assets/airlines/est.json

```diff
@@ -0,0 +1,17 @@
+{
+    "icao": "est",
+    "name": "Nordica",
+    "callsign": {
+        "name": "Reval",
+        "callsignFormats": [
+            "####"
+        ]
+    },
+    "fleets": {
+        "default": [
+            ["AT76", 2],
```

Comment:
These are not their whole fleet, but the others are operated for LOT or SAS, so I only included the ones operated under their own name.

Reply:
This airline doesn't actually seem to fly any flights for themselves, appears to have entered into partnerships with SAS and LOT, and their flights appear to all be operated for LOT/SAS - even their booking site redirects you to LOT! I would be tempted to add all their current fleet as listed at https://www.planespotters.net/airline/Xfly.

Reply:
I updated their fleet to exactly what is presented at the linked page. (see other comment as well)